CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+- Remove the ClusterClaim CRD and upgrade the ClusterSet CRD version to v1alpha2, and enhance the ClusterSet controller to support ClusterSet version upgrade. ([#5001](https://github.com/antrea-io/antrea/pull/5001) [#5250](https://github.com/antrea-io/antrea/pull/5250), [@luolanzone])
+- Increase the controller QPS setting in Multi-cluster Controller to improve multi-cluster resource export performance, and increase the LabelIdentity controller worker count to improve its performance. ([#5099](https://github.com/antrea-io/antrea/pull/5099), [@GraysonWu])
+- Change default port range to `40000-41000` for NodePortLocal on Windows to avoid race condition when
+inserting Netnat rules. ([#5107](https://github.com/antrea-io/antrea/pull/5107), [@XinShuYang])
+- Improve direct connections to Antrea apiserver in antctl with accessibility to Node ExternalIP and add a new `--insecure` option to support both secure and insecure connections. ([#5135](https://github.com/antrea-io/antrea/pull/5135), [@antoninbas])
+- Add two new fields, the "direction" of the NP rule (Ingress or Egress) and the reference of the Pod (<Namespace>/<Name>) to which the NP rule is applied, to audit logs. ([#5101](https://github.com/antrea-io/antrea/pull/5101), [@antoninbas])
+- Add a FlowExporter configuration toggle to antrea-agent for users to explicitly enable/disable flow exports. ([#5021](https://github.com/antrea-io/antrea/pull/5021), [@yuntanghsu])
+- Add OpenAPI schema for the AntreaAgentInfo and AntreaControllerInfo CRDs. ([#5206](https://github.com/antrea-io/antrea/pull/5206), [@ceclinux])
+- Add a validation schema for the matchLabels field of the ExternalIPPool CRD. ([#5284](https://github.com/antrea-io/antrea/pull/5284), [@tnqn])
+- Update short-names for AntreaNetworkPolicy as ANNP. ([#5081](https://github.com/antrea-io/antrea/pull/5081), [@qiyueyao])
+- Disable CGO for the antctl binary to avoid GLIBC dependency for cross-platform compatibility. ([#5240](https://github.com/antrea-io/antrea/pull/5240), [@XinShuYang])
+- Change the sftp image container registry to `projects.registry.vmware.com`. ([#5197](https://github.com/antrea-io/antrea/pull/5197), [@ceclinux])
+- Use a new API in ofnet to remove the unnecessary map to reduce agent memory usage. ([#5054](https://github.com/antrea-io/antrea/pull/5054), [@wenyingd])
+- Use syscall to query or operate network adapters on Windows to reduce operation delay. ([#4898](https://github.com/antrea-io/antrea/pull/4898), [@wenyingd] [@qiyueyao])
+- Update out-of-date audit logs docs for new log fields. ([#5199](https://github.com/antrea-io/antrea/pull/5199), [@cr7258])
+- Switched to strctured logging and change verbosity of potentially misleading Info log in the Antrea NetworkPolicy reconciler. ([#5048](https://github.com/antrea-io/antrea/pull/5048), [@antoninbas])
```

Comment:
typo ?
should be "Switched to structured logging"

Reply:
Fixed.